how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,320 @@
+
+As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB instances, 3 TiKV instances, and 3 PD instances. With TiUP Playground, you can quickly build the above test cluster by taking the following steps:
+
+1. Download and install TiUP:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. Declare the global environment variable:
+
+    {< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ``` 
+
+    > **Note:**
+    >
+    > After the installation, TiUP gives the absolute path of the corresponding `profile` file. You need to modify the above `source` command according to the path.
+
+3. Install TiUP Playground:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup install playground
+    ```
+
+    If you have installed TiUP Playground before, execute the following command to upgrade it to the latest version:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup update playground
+    ```
+
+4. Execute the following command in the current session to start the cluster:
+
+    - If you want to start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, and 1 PD instance, run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground
+        ```
+    
+    - If you want to specify the version and the number of the component instances, run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground v4.0.0-rc --db 2 --pd 3 --kv 3 --monitor
+        ```
+    
+        The command downloads a v4.0.0-rc cluster to the local machine and starts it.
+
+        `--monitor` means that the monitoring component will also be deployed.
+
+        The execution results show the access methods of the cluster:
+
+        ```log
+        CLUSTER START SUCCESSFULLY, Enjoy it ^-^
+        To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+        To connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+        To view the dashboard: http://127.0.0.1:2379/dashboard
+        To view the monitor: http://127.0.0.1:9090
+        ```
+
+5. Start a new session to access TiDB:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    mysql --host 127.0.0.1 --port 4000 -u root
+    ```
+
+6. Access the Prometheus dashboard of TiDB at <http://127.0.0.1:9090>.
+
+7. Access the TiDB Dashboard at <http://127.0.0.1:2379/dashboard>. The default username is `root`, with an empty password.
+
+8. After the test, you can clean the cluster by taking the following steps:
+
+    1. Stop the process by pressing `ctrl-c`.
+
+    2. Run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup clean --all
+        ```
+
+## Deploy a quasi-production test environment on a single machine using TiUP cluster
```

Comment:
```suggestion
## Set up a quasi-production environment on a single machine using TiUP cluster
```

Reply:
I am not sure of the statement of "quasi-production environment", because a single-machine cluster is hardly close to production deployment. I know the Chinese version says so, but in English, let's just use `test` environment.